lib/ui/views/dashboard_view.dart

```diff
@@ -167,6 +162,47 @@ class _DashboardViewState extends State<DashboardView>
         ]),
       );
 
+  Widget _buildProgressBarCard(
+          DashboardViewModel model, PreferencesFlag flag) =>
+      DismissibleCard(
+        isBusy: model.busy(model.progress),
+        key: UniqueKey(),
+        onDismissed: (DismissDirection direction) {
+          dismissCard(model, flag);
+        },
+        child: Column(mainAxisSize: MainAxisSize.min, children: [
+          Align(
+              alignment: Alignment.centerLeft,
+              child: Container(
+                padding: const EdgeInsets.fromLTRB(17, 15, 0, 0),
+                child: Text(AppIntl.of(context).progress_bar_title,
+                    style: Theme.of(context).textTheme.headline6),
+              )),
+          Stack(children: [
+            Container(
+              padding: const EdgeInsets.fromLTRB(17, 10, 15, 20),
+              child: ClipRRect(
+                borderRadius: const BorderRadius.all(Radius.circular(10)),
+                child: LinearProgressIndicator(
+                  value: model.progress,
+                  minHeight: 30,
+                  valueColor: const AlwaysStoppedAnimation<Color>(
```

Comment:
I agree with @camillebrulotte idea to change the color of the text, I think we could also match the theme better by showing the progress in red, using ` valueColor: const AlwaysStoppedAnimation<Color>(AppTheme.accent)` on (l.180) this is what it looks like:
<details>
  <summary>Click to expand 😄 </summary>

![image](https://user-images.githubusercontent.com/66817481/119287443-7f893b80-bc14-11eb-8870-d272166092e6.png)

![image](https://user-images.githubusercontent.com/66817481/119287476-92037500-bc14-11eb-97ad-321c1e712fad.png)

</details>

Reply:
hmm je comprends ton point, mais rouge fait beaucoup "Danger" et ça capte beaucoup l'attention je trouve dans ce cas là, tandis que vert est un peu plus soft et ça match aussi la couleur dans les notes :D

Reply:
Je suis d'avis pour garder le vert aussi (plus joyeux haha)

Reply:
Alright, lets keep it green then 😄